infinex-patron/patron-purchase-app/testnets/infinex-patron-test-add-app.toml

```diff
@@ -0,0 +1,24 @@
+name = "infinex-app-registration"
+version = "1.0.0"
+
+# pnpm cannon build infinex-patron/patron-purchase-app/testnets/infinex-patron-test-add-app.toml --chain-id 421614 --dry-run
+# pnpm cannon build infinex-patron/patron-purchase-app/testnets/infinex-patron-test-add-app.toml --chain-id 11155111 --dry-run
+# pnpm cannon build infinex-patron/patron-purchase-app/testnets/infinex-patron-test-add-app.toml --chain-id 80002 --dry-run
+# pnpm cannon build infinex-patron/patron-purchase-app/testnets/infinex-patron-test-add-app.toml --chain-id 11155420 --dry-run
+# pnpm cannon build infinex-patron/patron-purchase-app/testnets/infinex-patron-test-add-app.toml --chain-id 84532 --dry-run
+
+[pull.latestVersion]
+source = "infinex-core-test:3.0.1@core"
```

Comment:
this should be 
```suggestion
source = "infinex-core-test:latest@core"
```

Reply:
I disagree, we should always be specific about versions

Reply:
this is for testnet, and you want to pull the latest version, as indicated by the name. there's a high chance you forget to update the version number when you run this script, and end up with a misconfiguration

Reply:
and an even greater risk that "latest" gets copied into a prod file because the precedent was set.
@lemoninfinex ?

Reply:
if the specific version gets copied into prod file then you end up with the same problem where the wrong version was used, and I would argue having it set to latest is safer, because that should always be the right deployment